samcli/hook_packages/terraform/Config.json

```diff
@@ -0,0 +1,16 @@
+{
+    "hook_package_id": "terraform",
+    "description": "Terraform IaC hooks package",
+    "hook_use_case": "IaC",
+    "iac_framework": "Terraform",
+    "version": "1.0",
+    "hook_specification": "1.0.0",
+    "functionalities": {
+         "prepare": {
+              "entry_method": {
+                    "module": "samcli.hook_packages.terraform.prepare.hook",
+                    "method": "hook"
```

Comment:
wasn't very creative here lol, open to suggestions

Reply:
method name can be `prepare`, `deploy` .. etc